### PR TITLE
KAFKA-17711: Minor cleanup changes in ShareConsumeRequestManager

### PR DESCRIPTION
**Pull request by ShivsundarR**
*What*
Minor cleanup and javadoc changes in ShareConsumeRequestManager.

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java [hunk 1]
```diff
@@ -284,6 +285,15 @@ private void maybeSendShareAcknowledgeCommitCallbackEvent(Map<TopicIdPartition,
         }
     }
 
+    /**
+     *
+     * @param acknowledgeRequestState Contains the acknowledgements to be sent.
+     * @param currentTimeMs The current time in ms.
+     * @param onCommitAsync Boolean to denote if the acknowledgements came from a commitAsync or not.
+     * @param isAsyncDone Boolean to indicate if the async request has been sent.
```
Comment (AndrewJSchofield) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java]: Should this one be called `isAsyncSent` too?

**Review by AndrewJSchofield — CHANGES_REQUESTED** [review]
Thanks for the PR. Just one minor comment.

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java (hunk 1 above)
Comment (ShivsundarR) [reply clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java]: Yeah makes sense, I have updated the PR. Thanks.

**Review by AndrewJSchofield — APPROVED** [review]

**Review by omkreddy — APPROVED** [review]
@ShivsundarR Thanks for the PR. LGTM

**Pull request merged**